src/content/platform-integration/ios/platform-views.md

```diff
@@ -174,10 +174,10 @@ import UIKit
     ) -> Bool {
         GeneratedPluginRegistrant.register(with: self)
 
-        weak var registrar = self.registrar(forPlugin: "plugin-name")
+        weak var pluginRegistrar = self.registrar(forPlugin: "plugin-name")
 
-        let factory = FLNativeViewFactory(messenger: registrar!.messenger())
-        self.registrar(forPlugin: "<plugin-name>")!.register(
+        let factory = FLNativeViewFactory(messenger: pluginRegistrar!.messenger())
+        pluginRegistrar?.register(
```

Comment:
nit: can be `guard let pluginRegisterar = self.registrar(forPlugin: "plugin-name") else { return false }`

Reply:
Nice idea ;)
I didn't suggest this change before because I thought the `weak var` was needed to avoid some sort of retain cycle. In this example it's not possible to happen so I'll change to a `guard let`.

Done.